Cargo.toml

```
@@ -15,14 +15,19 @@ flate2 = "1.0.20"
 futures = "0.3.31"
 jpeg = { package = "jpeg-decoder", version = "0.3.0", default-features = false }
 num_enum = "0.7.3"
-object_store = "0.12"
+object_store = { version = "0.12", optional = true }
 # In the future we could make this feature-flagged, but for now we depend on
 # object_store which uses reqwest.
-reqwest = { version = "0.12", default-features = false }
+reqwest = { version = "0.12", default-features = false, optional = true }
 thiserror = "1"
 tokio = { version = "1.43.0", optional = true }
 weezl = "0.1.0"
 
 [dev-dependencies]
 tiff = "0.9.1"
 tokio = { version = "1.9", features = ["macros", "fs", "rt-multi-thread"] }
+
+[features]
+default = ["object_store"]
+reqwest = ["dep:reqwest"]
+object_store = ["dep:object_store", "reqwest"]
```

```suggestion
default = ["object_store", "reqwest"]
reqwest = ["dep:reqwest"]
object_store = ["dep:object_store"]
```
In the future `object_store` may not depend on `reqwest`